README.md

```diff
@@ -25,6 +25,41 @@ In your app delegate's `application:didFinishLaunchingWithOptions:` method, init
 
 Tryouts SDK will automatically check for new releases every time the app becomes active and alert users when there is a new version available.
 
+### Feedback
+
+Tryouts SDK can be used to get feedbacks from users of an application. The method to be used is `presentFeedBackControllerWithPresentingViewController:animated:`
```

Comment:
"Tryouts SDK can be used to gather in-app feedback from your users"

Also update the example method name here according to the change above